shared/ff_util/include/ff_util/ff_service.h

```diff
@@ -100,18 +91,17 @@ class FreeFlyerServiceClient {
     return false;
   }
 
-  bool Call(FreeFlyerService<ServiceRequestSpec, ServiceResponseSpec> &
-                                                                    service) {
-    return call(service);
+  bool Call(const typename ServiceSpec::Request & request, std::shared_ptr<typename ServiceSpec::Response> & response) {
+    return call(request, response);
   }
 
   // ROS1 functions
-  bool call(FreeFlyerService<ServiceRequestSpec, ServiceResponseSpec>& service) {
+  bool call(const typename ServiceSpec::Request & request, std::shared_ptr<typename ServiceSpec::Response> & response) {
```

Comment:
@kbrowne15 I thought it was easier this way. Let me know if you disagree

Reply:
Um, so I wonder if we could overload it and do it both ways. The executive, which has 24 service clients, used a service object instead of a request and a response so I was trying to minimize the code changes there. The behaviors also seem to use the service object.

Reply:
oh, I didn't know this was used elsewhere. yeah of course.